eksupgrade/src/eks_get_image_type.py

```diff
@@ -16,31 +16,18 @@ def image_type(node_type: str, image_id: str, region: str) -> Optional[str]:
     """Return the image location."""
     ec2_client = boto3.client("ec2", region_name=region)
     node_type = node_type.lower()
+    filters = [
+        {"Name": "is-public", "Values": ["true"]},
+    ]
 
     if node_type == "amazon linux 2":
-        filters = [
-            {"Name": "owner-id", "Values": ["602401143452"]},
-            {"Name": "name", "Values": ["amazon-eks-node-*"]},
-            {"Name": "is-public", "Values": ["true"]},
-        ]
+        filters.append({"Name": "name", "Values": ["amazon-eks-node-*"]})
```

Comment:
We should change this to pull from the SSM parameters since this is the recommended approach. There may exist a newer AMI available based on these filters, but it may not be meant for consumption for various reasons (if the AMI is being recalled, etc.)